apps/sendsignal.go

```diff
@@ -0,0 +1,29 @@
+package apps
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+)
+
+func SendSignal(ctx context.Context, app string, signal string, args []string) error {
```

Comment:
best practice: I don't like the `args` variable name. It's not at all descriptive of what it should contain.